### PR TITLE
Little improvements

### PR DESCRIPTION
**Pull request by heinwessels**
Hello,

I saw a streamer using your mod and I really liked how intuitively it works with the mapview. There was a few small issues that I was mentioned so I thought I'd fix them for you, if you don't mind :)

I also added a simple thumbnail and changelog, although I didn't try to add in the changes in previous versions. Now the changelog will show in the mod portal and in-game.

### CODE REVIEW

File: gui.lua [hunk 1]
```diff
@@ -18,7 +18,7 @@ end
 
 local function get_tag_caption(tag)
     str = signal_id_to_rich_text(tag.icon, " [img=utility/custom_tag_in_map_view] ") ..
-                " " .. util.trim(tag.text:sub(config.tag_prefix_len + 1):gsub(util.ASSIGNEE_PATTERN, "[color=blue]@%1[/color]"))
+                " " .. util.trim(tag.text:gsub("TODO:?", ""):gsub(util.ASSIGNEE_PATTERN, "[color=blue]@%1[/color]"))
```
Comment (Doridian) [inline gui.lua]: I need to read up on my Lua again, but wouldn't that replace `TODO` at any point in the message, not just the start?

**Comment by Doridian** [comment]
First of all, thanks for your PR. Most of it looks great. Also the thumbnail....I really should've gotten around to do that.

File: gui.lua (hunk 1 above)
Comment (heinwessels) [reply gui.lua]: Ah yeah, that's not good. Luckily was an easy fix.

**Comment by heinwessels** [comment]
> First of all, thanks for your PR. Most of it looks great. Also the thumbnail....I really should've gotten around to do that.

No worries. And life gets busy fast, can't do everything.

**Pull request merged**